src/main/java/org/jabref/model/openoffice/uno/DocumentAnnotation.java

```diff
@@ -0,0 +1,15 @@
+package org.jabref.model.openoffice.uno;
```

Comment:
Move the class one level up. It just consists of classes residing in package `com.sun.star.text` and therefore is not directly part of `uno`, is it?

If it is, add a small comment to the package to explain why the class resides here.

(Your survery asked about that and therefore I am thinking more about it than I should :p)

Reply:
https://devdocs.jabref.org/code-howtos/openoffice/code-reorganization.html
uno : helpers for various tasks via UNO.
These are conceptually independent of JabRef code and logic.